docs/api/solvers/sde_solvers.md

```diff
@@ -113,3 +113,43 @@ These are reversible in the same way as when applied to ODEs. [See here.](./ode_
     selection:
         members:
             - __init__
+
+
+---
+
+### Underdamped Langevin solvers
+
+These solvers are specifically designed for the Underdamped Langevin diffusion (ULD),
+which takes the form 
+
+\begin{align*}
+    \mathrm{d} x(t) &= v(t) \, \mathrm{d}t \\
+    \mathrm{d} v(t) &= - \gamma \, v(t) \, \mathrm{d}t - u \,
+    \nabla \! f( x(t) ) \, \mathrm{d}t + \sqrt{2 \gamma u} \, \mathrm{d} w(t),
+\end{align*}
```

Comment:
Does this get rendered correctly in the docs? It might need some surrounding `$`?

Reply:
It does for me. Do you think I should still try to use `$`?

Reply:
As long as it doe! I've not tried myself.